modules/tinymce/src/core/test/ts/browser/textpatterns/ReplacementTest.ts

```diff
@@ -150,4 +152,38 @@ describe('browser.tinymce.core.textpatterns.ReplacementTest', () => {
     // This function throws an error here due to the bug identified in TINY-8779
     TinyContentActions.keystroke(editor, Keys.enter());
   });
+
+  context('TINY-8779: Matches text nodes in a paragraph', () => {
+    let editor: Editor;
```

Comment:
This shouldn't be done as we try to avoid mutable variables. Instead we should just either use the `hook.editor()` call in the `setCursorAndPressEnterAndAssert` function or pass it through in the calls beforehand.